easybuild/easyconfigs/l/libpciaccess/libpciaccess-0.18.1-GCCcore-14.2.0.eb

```
@@ -0,0 +1,27 @@
+easyblock = 'MesonNinja'
+
+name = 'libpciaccess'
+version = '0.18.1'
+
+homepage = 'https://cgit.freedesktop.org/xorg/lib/libpciaccess/'
+description = """Generic PCI access library."""
+
+toolchain = {'name': 'GCCcore', 'version': '14.2.0'}
+
+source_urls = ['https://www.x.org/releases/individual/lib/']
+sources = [SOURCE_TAR_XZ]
+checksums = ['4af43444b38adb5545d0ed1c2ce46d9608cc47b31c2387fc5181656765a6fa76']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('xorg-macros', '1.20.2'),
+    ('Meson', '1.6.1'),
+    ('Ninja', '1.12.1'),
+]
+
```

```suggestion

configopts = "--default-library=both"  # static and shared library

```

Should we maintain generating the static as well?

---

This also matches https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/l/libpciaccess/libpciaccess-0.18.1-GCCcore-13.3.0.eb#L22